### DISCUSSION TITLE
Nova getting 403 Forbidden

### DISCUSSION THREAD

**Discussion by marosjev-cde** [deepgram/community#301] [General help]
### Which Deepgram product are you using?

Deepgram API

### Details

Hello,

I would like to ask if I am making some kind of mistake somewhere while using Nova model for transcribing a prerecorded audio file.

line 98: <class ‘Exception’> - DG: 403, message=‘Forbidden’, url=URL(‘https://api.deepgram.com/v1/listen?diarize=true&punctuate=true&tier=nova&model=general’)

When using base or enhanced tier, I have no problems.

Thank you.

Best,
Maros J

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

https://api.deepgram.com/v1/listen?diarize=true&punctuate=true&tier=nova&model=general

**Comment by jjmaldonis** [comment]
Hi Maros, that's very strange, those query parameters should not result in a permission denied error.

I have two questions that will help track this down:

1. Do you have a sales contract with Deepgram?
2. Do you know when you created your account?

**Reply by marosjev-cde** [reply]
Hello Jason,

1. I am using free version at the moment for some bench-marking.
2. February 2023

**Comment by marosjev-cde** [comment]
Hello Jason,

I also tried to run it on playground but got the similar error in there:

err_code: "UNAUTHORIZED_METHOD_TIER_REQUESTED"
err_msg: "Project does not have access to the requested processing method or model tier."
request_id: "7af2660b-23ef-4dfa-99ee-6085a1926d42"

Is there a limitation that Nova tier is only allowed for premium users?

**Comment by jjmaldonis** [comment]
We resolved this over email so I'm going to close this discussion.
